release/archives/build.gradle

```diff
@@ -128,6 +129,7 @@ subprojects {
             def tarWithJDKTask = tasks.getByName("${platformWithArchitecture}WithJDKDistTar")
             def zipTask = tasks.getByName("${platformWithArchitecture}DistZip")
             def zipWithJDKTask = tasks.getByName("${platformWithArchitecture}WithJDKDistZip")
+            def destinationKeyPath = "${project.rootProject.version}/${project.buildNumber}/archive"
```

Comment:
How will any retry with same build number handled ?

Reply:
Presently, this would overwrite the existing number. GitHub Actions does provide for unique build numbers here, so I don't think it is a very likely scenario.

Also, the current approach definitely overwrites since there is no build number at all.